docs/guides/cloud/data-storage-and-masking.mdx

```diff
@@ -0,0 +1,169 @@
+---
+title: Data Storage and Controls
+sidebar_position: 60
+sidebar_label: Data Storage and Controls
+---
+
+:::info
+
+## <Icon name="graduation-cap" /> What you'll learn
+
+- What data does Cypress Cloud store?
+- What controls exist to control data sent to and viewed in Cypress Cloud?
+
+:::
+
+## Data Storage
+
+:::caution
+
+Please refer to the [Cypress Cloud Terms of Use](https://cloud.cypress.io/terms-of-use) and our [Security & Compliance](https://www.cypress.io/security/) guide for more specifics around reasonable use of Cypress Cloud, data storage and security at Cypress.
+
+:::
+
+When you run Cypress via `cypress run` passing the `--record` flag the following data is stored in Cypress Cloud for every run:
+
+- Standard output in the terminal
+- Test results
+- Test definitions
+- Cypress configuration (minus Cypress environment variables)
+- Screenshots
+- Videos
+- OS environment variables related to CI and git information
+
+When [Test Replay](#Test-Replay) capturing is enabled, the additional data outlined below will be sent to the Cloud. You can disable capturing: TODO link this data.
```

Comment:
Should this be linked to the main TR product page?

Reply:
also the disable portion will link to `/guides/cloud/debugging/test-replay#Opt-out-of-Test-Replay`

Reply:
We refer to what's stored in Cypress Cloud in the first list above, but just call it Cloud here. Could go either way depending on the context, but we should make them consistent.

Reply:
Address Cypress Cloud part in [`0b22fd0` (#5425)](https://github.com/cypress-io/cypress-documentation/pull/5425/commits/0b22fd0a871b2d32d1f79d79fcb4b2fc51ee4d08)

Yah, a lot of this is hard to do without having the Test Replay merged into the v13 branch. I'll have to update the links after.